### DISCUSSION TITLE
NPM (nginx proxy manager) "map" directive is not allowed here for authentik nginx config

### DISCUSSION THREAD

**Discussion by SpiderUnderUrBed** [goauthentik/authentik#10004] [Q&A]
Hi, I am trying to protect my applications behind authentik, but the server just becomes offline when i add the recommended NPM configuration into the advanced  section inside NPM
Here is the config:
https://pastebin.com/sUHxErzH
Here is the error:

```
[root@docker-6c71d0ef7573:/var/log/nginx]# cat error.log
2024/06/06 04:01:08 [emerg] 206#206: "map" directive is not allowed here in /data/nginx/proxy_host/3.conf:47
2024/06/06 04:01:09 [notice] 210#210: signal process started
```

**Comment by phulstaert** [comment]
It is a config error.
You cannot put the map command in the 'custom nginx configuration' textbox. There is a relatively easy way out of this.

See [here](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3474#issuecomment-2145962081)
